torchquad/tests/boole_test.py

```diff
@@ -61,6 +61,65 @@ def _run_boole_tests(backend, _precision):
     # for error in errors:
     # assert error < 5e-9
 
+    # JIT Tests
+    if backend != "numpy":
+        N = 401
+        jit_integrate = None
+
+        def integrate(*args, **kwargs):
+            nonlocal jit_integrate
+            if jit_integrate is None:
```

Comment:
Could you add a small comment explaining the though process here? It is a bit confusing :thinking:

Reply:
done (i think)